packages/commerce-sdk-react/src/public-utils.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I know I already reviewed this.. but I think it's a little abnormal. Yes we can't make thing private by default because we are using compilation in a way that doesn't result in a single file. But typically when you are using naming to declare a files protection level its only done for private things.. like files that are private starting with __ or folders marked internal. I feel like in this instance we are doing the opposite.

We could do 2 things. 

1. Name this normally as everything is currently already public. 
2. Make use of the "exports" field in the package.json
3. Overhaul the compilation to return a single es6 lib (I know we aren't going to do that right now, but I had to mention it)

Reply:
I'll move this back to `utils.ts` file.

Reply:
done